src/lime/media/OpenALAudioContext.hx

```diff
@@ -91,6 +91,13 @@ class OpenALAudioContext
 	public var DEFAULT_ALL_DEVICES_SPECIFIER:Int = 0x1012;
 	public var ALL_DEVICES_SPECIFIER:Int = 0x1013;
 
+	public var DEVICE_CLOCK_SOFT:Int = 0x1600;
+	public var DEVICE_LATENCY_SOFT:Int = 0x1601;
+	public var DEVICE_CLOCK_LATENCY_SOFT:Int = 0x1602;
+
+	public var SEC_OFFSET_LATENCY_SOFT:Int = 0x1201;
+	public var SEC_OFFSET_CLOCK_SOFT:Int = 0x1203;
+
```

Comment:
Ugh, why are we even storing ~90 constants as variables? (Not your fault/responsibility, this PR just made me aware of it.)

Is it so that `openfl.media.SoundChannel` can access it as an instance variable? That implies it might be overridden, like via an alternative AL implementation. But the accesses only happen within `#if lime_openal`, so it wouldn't even attempt to get them if it wasn't OpenAL.

Reply:
I took a closer look, and this file's list was always incomplete. Perhaps intentionally? In any case there's no need for this PR to add to it.